jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-client/src/main/java/org/jbpm/console/ng/pr/client/editors/instance/signal/ProcessInstanceSignalViewImpl.html

```diff
@@ -1,26 +1,26 @@
-<div class="container-fluid">
-    <div class="form-horizontal">
-        <fieldset>
-            <div class="form-group">
-                <label class="control-label col-md-2" for="signalRefText" data-field="signalRefLabel"></label>
+<div>
+    <div>
+        <div class="form-horizontal">
+            <fieldset>
+                <div class="form-group">
+                    <label class="control-label col-md-3" for="signalRefText" data-field="signalRefLabel"></label>
 
-                <div class="col-md-10">
-                    <input type="text" data-field="signalRefText" id="signalRefText" placeholder="" class="form-control"/>
+                    <div class="col-md-9">
+                        <input type="text" data-field="signalRefText" id="signalRefText" placeholder="" class="form-control"/>
```

Comment:
just a note. it's not required to use a data-field, just id is fine. @Inject @DataField will work for id, data-field, and class.